tests/integration/targets/vmware_guest/defaults/main.yml

```diff
@@ -19,6 +19,7 @@ vmware_guest_test_playbooks:
   - mac_address_d1_c1_f0.yml
   - max_connections.yml
   - mem_reservation.yml
+  - multiple_disk_controllers_d1_c1_f0.yml
```

Comment:
+1, hopefully it won't blow things up.

Reply:
@Akasurde @goneri I think this is yet another problem with the vCenter simulator. I have moved those tests into a `when: vcsim is not defined` block.

Btw: It looks like we don't test against a real vCenter in the CI pipeline anymore. Would it be possible to get this back? It's really unfortunate that there are so many integration tests that are skipped because we only have vcsim.